ynput/ayon-unreal
# Camera Publisher: Small tweaks on extractors due to unreal version and validators to check on the camera in level sequence

## Changelog Description
Resolve #63 

## Additional info
n/a


## Testing notes:

1. Select the Level Sequence in the content browser you want to publish (And make sure that level sequence includes the track from the camera of which you want to publish)
![image](https://github.com/user-attachments/assets/e671f31a-3e1a-4ba0-a877-38180709ef19)

2. Create Camera Instance
3. Publish
4. If there is no camera cut tracks inside the level sequence, it will error out.
5. You can perform repair action to add camera cut track but you need to assign the target camera you want to publish by your own.
